selector.go

```diff
@@ -106,24 +107,35 @@ func FilterIn(values ...string) *SimpleFilter {
 func FilterNotIn(values ...string) *SimpleFilter {
 	fs := make([]*SimpleFilter, 0, len(values))
 	for _, v := range values {
-		fs = append(fs, FilterNE(v))
+		f := FilterNE(v)
+		fs = append(fs, f)
```

Comment:
One-liner looked cleaner to me x2

Reply:
https://github.com/nspcc-dev/netmap/pull/14/commits/4b185d39e510f485533f35a1e1c132f2a0208f16